charts/hyades/README.md

```diff
@@ -29,259 +29,261 @@ The GA roadmap for Hyades is tracked here: https://github.com/DependencyTrack/hy
-| vulnAnalyzer.image.registry | string | `""` | Override common.image.registry for the vulnerability analyzer. |
-| vulnAnalyzer.image.repository | string | `"dependencytrack/hyades-vulnerability-analyzer"` |  |
-| vulnAnalyzer.image.tag | string | `"snapshot-native"` | Can be a tag name such as "latest", or an image digest prefixed with "sha256:". |
-| vulnAnalyzer.initContainers | list | `[]` | Additional init containers to deploy. Supports templating. |
-| vulnAnalyzer.nodeSelector | object | `{}` |  |
-| vulnAnalyzer.persistentVolume.className | string | `""` |  |
-| vulnAnalyzer.persistentVolume.enabled | bool | `false` | Whether to use a persistent volume to store application state. Has no effect unless useStatefulSet is true. Is pointless unless the STATE_STORE_TYPE environment variable is set to "rocks_db", since state is kept in memory per default. A volume will be created for each Pod, via volumeClaimTemplates, since state must not be shared between multiple replicas. |
-| vulnAnalyzer.persistentVolume.size | string | `"2Gi"` |  |
-| vulnAnalyzer.probes.liveness.failureThreshold | int | `3` |  |
-| vulnAnalyzer.probes.liveness.initialDelaySeconds | int | `10` |  |
-| vulnAnalyzer.probes.liveness.periodSeconds | int | `15` |  |
-| vulnAnalyzer.probes.liveness.successThreshold | int | `1` |  |
-| vulnAnalyzer.probes.liveness.timeoutSeconds | int | `5` |  |
-| vulnAnalyzer.probes.readiness.failureThreshold | int | `3` |  |
-| vulnAnalyzer.probes.readiness.initialDelaySeconds | int | `10` |  |
-| vulnAnalyzer.probes.readiness.periodSeconds | int | `15` |  |
-| vulnAnalyzer.probes.readiness.successThreshold | int | `1` |  |
-| vulnAnalyzer.probes.readiness.timeoutSeconds | int | `5` |  |
-| vulnAnalyzer.replicaCount | int | `1` |  |
-| vulnAnalyzer.resources.limits.cpu | string | `"2"` |  |
-| vulnAnalyzer.resources.limits.memory | string | `"2Gi"` |  |
-| vulnAnalyzer.resources.requests.cpu | string | `"500m"` |  |
-| vulnAnalyzer.resources.requests.memory | string | `"512Mi"` |  |
-| vulnAnalyzer.securityContext | object | `{"allowPrivilegeEscalation":false,"capabilities":{"drop":["ALL"]},"readOnlyRootFilesystem":true,"runAsNonRoot":true,"seccompProfile":{"type":"RuntimeDefault"}}` | Security context of the container. |
-| vulnAnalyzer.service.annotations | object | `{}` |  |
-| vulnAnalyzer.tolerations | list | `[]` |  |
-| vulnAnalyzer.useStatefulSet | bool | `false` | Whether to deploy the vulnerability analyzer as StatefulSet. Should be enabled when the STATE_STORE_TYPE environment variable is set to "rocks_db", to allow for reliable storage of application state to disk. Should be used in combination with persistentVolume.enabled. |
-
+| Key                                                        | Type   | Default                                                                                                                                                            | Description |
```

Comment:
Here, everything is changing because I formatted the table with the proper ident. Much easier to read in the IDE:
![image](https://github.com/user-attachments/assets/afe5abee-d1e9-4329-bb12-62a1f3cfad9a)